pkg/rebuild/schema/schema.go

```diff
@@ -225,16 +226,12 @@ func NewRebuildPackageRequest(form url.Values) (*RebuildPackageRequest, error) {
 	if req.ID == "" {
 		return nil, errors.New("No ID provided")
 	}
-	if encStrat := form.Get("strategy"); encStrat != "" {
-		oneof := StrategyOneOf{}
-		err := json.Unmarshal([]byte(encStrat), &oneof)
+	if fromRepo := form.Get("strategyFromRepo"); fromRepo != "" {
+		strategyFromRepo, err := strconv.ParseBool(fromRepo)
 		if err != nil {
 			return nil, err
 		}
-		if _, err := oneof.Strategy(); err != nil {
-			return nil, err
-		}
-		req.StrategyOneof = &oneof
+		req.StrategyFromRepo = strategyFromRepo
```

Comment:
nit: Maybe just assign directly to req.StrategyFromRepo?

Reply:
it was gonna cost an extra line regardless since `err` isn't in scope. either we need a forward declaration or a re-assignment. didn't feel one was particularly better than the other.